docs/getting-started/who-supports-saf.md

```diff
@@ -44,9 +44,9 @@ See below overview tables which will give you an idea of who supports SAF and to
 | RelConnectsSurfaceEdge      |                                 Yes                                 |                                  Yes                                  |                                           Yes                                           |                                   No                                  |                                      Yes                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                          |                                          No                                          | Yes | No | No | No |
-| RelConnectsRigidCross       |                                  No                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                      Yes                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                           |                                          No                                          | Yes | No | No | No |
+| RelConnectsRigidCross       |                                  Yes                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                      Yes                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                           |                                          No                                          | Yes | No | No | No |
```

Comment:
These changes are not described in the issue #138, I've noticed there are obsolete information fro SCIA, so fixed them here.